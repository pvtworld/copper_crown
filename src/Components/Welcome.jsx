import React from 'react';
import {Link} from 'react-router-dom';

class Welcome extends React.Component {
    render() {
        return (
            <div>
                <h1>Welcome to CopperCrown!</h1>
                <div>
                    <Link to="/login">Register/Login</Link>
                </div>
            </div>
        )
    };
}

export default Welcome;