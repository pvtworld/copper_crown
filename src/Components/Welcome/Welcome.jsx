import React from 'react';
import {Link} from 'react-router-dom';

export default class Welcome extends React.Component {
    render() {
        console.log(this.props)
        return (
            <div>
                <h1>Welcome to CopperCrown!</h1>
                <div>
                    <Link to="/login">Register/Login</Link>
                </div>
            </div>
        );
    }
};