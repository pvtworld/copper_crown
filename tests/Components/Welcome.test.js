import React from 'react';
import ReactDOM from 'react-dom';
import Welcome from '../src/Components';

beforeAll(() => {
    
});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(, div);
});