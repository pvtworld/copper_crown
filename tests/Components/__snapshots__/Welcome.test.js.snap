exports[`Render test Renders without crashing 1`] = `
<div>
  <h1>
    Welcome to CopperCrown!
  </h1>
</div>
`;
